src/hypothesis/strategies.py

```diff
@@ -804,9 +804,8 @@ def dm_func(denom):
 
 @cacheable
 @defines_strategy
-def decimals(
-    min_value=None, max_value=None, allow_nan=None, allow_infinity=None
-):
+def decimals(min_value=None, max_value=None, allow_nan=None,
+             allow_infinity=None, places=None, precision=None):
```

Comment:
But likewise difficult to move from bounds to a single argument, I think - how about we defer this bit of design to @DRMacIver?